src/lib/Sympa/Request/Handler/del.pm

```diff
@@ -126,7 +126,8 @@ sub _report_member {
                 $who);
         }
     }
-    $self->add_stash($request, 'notice', 'removed', {'email' => $who});
+    $self->add_stash($request, 'notice', 'removed',
+        {'email' => $who, 'listname' => $list->get_id});
```

Comment:
Just for information: `listname` is used in `default/mail_tt2/report.tt2` but was not defined, so I added it (I noticed it while using the new `bouncers del _list_` feature).